bigbluebutton-html5/imports/ui/components/audio/service.js

```diff
@@ -69,7 +69,9 @@ export default {
   joinEchoTest: () => AudioManager.joinEchoTest(),
   toggleMuteMicrophone,
   changeInputDevice: inputDeviceId => AudioManager.changeInputDevice(inputDeviceId),
-  changeOutputDevice: outputDeviceId => AudioManager.changeOutputDevice(outputDeviceId),
+  changeOutputDevice: (outputDeviceId) => {
+    if (!AudioManager.initialized)AudioManager.changeOutputDevice(outputDeviceId);
```

Comment:
Why do we check for this condition? Why only for Output device and not input for example?

Reply:
Our output device is an audio tag on the client, but our input device are devices that are chosen by the user

Reply:
https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/audio/container.jsx#L141

Reply:
Still not sure about this. Checking with @capilkey